seq2seq/third_party/trainers/t5_trainer.py

```diff
@@ -1,7 +1,24 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
```

Comment:
Since this looks like a heavily modified version of the original HuggingFace file, we need to indicate that it has been modified here, as described in the doc I sent you.

Reply:
thanks, done